docs/reference/commandline/container_run.md

```diff
@@ -1000,7 +1000,7 @@ text. However, if you try the same thing but also adding the `-t` flag, the
 password is hidden:
 
 ```console
-$ docker run -i debian passwd root
+$ docker run -ti debian passwd root
```

Comment:
bikeshed / nit: should we converge on using `-it` as order for these? Checking what we use generally;


```bash
git grep '\-ti ' **/*.md | wc -l
       2
git grep '\-it ' **/*.md | wc -l
      79
```